src/pages/test/index.tsx

```
@@ -0,0 +1,314 @@
+import React, { useState } from 'react';
+import { useTrailPathsSafe, useTrailPaths } from '../../hooks/useTrailPaths';
```

ESLint

[error] 'useTrailPaths' is defined but never used.

(@typescript-eslint/no-unused-vars)